test/layer_tests.jl

```diff
@@ -3,71 +3,54 @@ using ChemistryFeaturization
 using SimpleWeightedGraphs
 
 include("../src/layers.jl")
-using .layers: AGNConv, AGNPool
+using .Layers: AGNConv, AGNPool
 
 @testset "AGNConv" begin
     # create simple line graph, populate it with feature of all ones
     adjmat = Float32.([0 1 0; 1 0 1; 0 1 0])
-    dummyfzn = [AtomFeat(:feat, [0]) for i = 1:4]
-    ag = AtomGraph(
-        SimpleWeightedGraph{Int32,Float32}(adjmat),
-        ["C", "C", "C"],
-        ones(Float32, 4, 3),
-        dummyfzn,
-    )
+    ag = AtomGraph(adjmat, ["C", "C", "C"])
+    dummyfzn = GraphNodeFeaturization(["Block"])
+    fa = featurize(ag, dummyfzn)
+
     # create a conv layer, initialize weights with ones
     l = AGNConv(4 => 4, initW = ones)
 
     # test output looks as expected
-    output_fea = l(ag).features
+    output_fea = l(fa)[2]
```

Comment:
Would it be easier (and more readable) if we created a convenience function for `FeaturizedAtoms`  that returned the encoded features?
Something like
```julia
 encodedfeatures(fa::FeaturizedAtoms) = fa.encoded_features
(or) 
 encoded_features(fa::FeaturizedAtoms) = fa.encoded_features
 ```
 maybe?

Reply:
I'm not sure that would actually help here since the layer doesn't return a `FeaturizedAtoms`. But as a separate concept, it's not a bad idea!